python-sdk/nuscenes/eval/panoptic/evaluate.py

```diff
@@ -240,12 +242,16 @@ def evaluate_tracking(self) -> Dict[str, Any]:
                     break
                 cur_token = cur_sample['next']
 
+        pat, mean_pq, mean_tq = self.evaluator['tracking'].get_pat()
         mean_ptq, class_all_ptq, mean_sptq, class_all_sptq = self.evaluator['tracking'].get_ptq()
         mean_iou, class_all_iou = self.evaluator['tracking'].getSemIoU()
         lstq, s_assoc = self.evaluator['tracking'].get_lstq()
         mean_motsa, mean_s_motsa, mean_motsp = self.evaluator['tracking'].get_motsa()
 
-        results = self.wrap_result_mopt(mean_ptq=mean_ptq,
```

Comment:
update the function doc-string part at line 203-210 to add the added PAT, PQ, TQ fields